<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>775</width>
    <height>563</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>280</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="tabletTracking">
    <bool>false</bool>
   </property>
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(9, 130, 230);
background-color: qlineargradient(spread:repeat, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(9, 130, 230 ,10) stop:1 rgba(255, 255, 255, 255));
border-radius:20px;
font-family: Lexend;
border-radius:5px;
color:#001F54;  /* Navy Blue */

font-size:20px;
border-color: #001F54;  /* Navy Blue */

border-style:20px solid #0982e6;;


QLineEdit
{
background:transparent;
border:none;
color:#0982e6;
border.bottum:1px solid #0982e6;
border-radius:25px;
}</string>
   </property>
   <property name="text">
    <string>Mot de passe </string>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::EchoMode::PasswordEchoOnEdit</enum>
   </property>
   <property name="dragEnabled">
    <bool>false</bool>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="cursorMoveStyle">
    <enum>Qt::CursorMoveStyle::LogicalMoveStyle</enum>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>70</y>
     <width>141</width>
     <height>131</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../—Pngtree—user name password data (3).png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="motdepasseoublier">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>340</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="tabletTracking">
    <bool>false</bool>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::ContextMenuPolicy::DefaultContextMenu</enum>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="inputMethodHints">
    <set>Qt::InputMethodHint::ImhNone</set>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="midLineWidth">
    <number>4</number>
   </property>
   <property name="text">
    <string>Mot de passe oublie?</string>
   </property>
   <property name="textFormat">
    <enum>Qt::TextFormat::AutoText</enum>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
   <property name="openExternalLinks">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>141</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../logo (2).png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>775</width>
     <height>563</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::NoFrame</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="add_employee_btn_3">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>460</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
border-radius:25px;
color:#001F54;  /* Navy Blue */
font-family: Lexend;
font-size:16px;
border-color: #001F54;  /* Navy Blue */
border-width : 1.2px;
border-style:solid;
background-color:white;
background-color: qlineargradient(spread:repeat, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(9, 130, 230 ,10) stop:1 rgba(255, 255, 255, 255));
}
QPushButton:hover
{
background:white;
color:#001F54;  /* Navy Blue */
border-left:3px solid #00b521 ; 
border-color:none;
border-radius:25px;
background-color: qlineargradient(spread:repeat, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(9, 130, 230 ,10) stop:1 rgba(255, 255, 255, 255));
}
QPushButton:pressed
{
background:#013220;
color:#001F54;  /* Navy Blue */
border-radius:25px;
font-family: Lexend;
font-size:16px;
border-left:3.2px solid #1BA019 ; 
/*background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                stop: 0 #ffffff, stop: 1 #dcdcdc);*/}

</string>
   </property>
   <property name="text">
    <string>connexion</string>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/images/add-circle.png</normalon>
    </iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>200</width>
     <height>200</height>
    </size>
   </property>
   <property name="autoRepeatInterval">
    <number>100</number>
   </property>
  </widget>
  <widget class="QLabel" name="motdepasseoublier_2">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>360</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="tabletTracking">
    <bool>false</bool>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::ContextMenuPolicy::DefaultContextMenu</enum>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="inputMethodHints">
    <set>Qt::InputMethodHint::ImhNone</set>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="midLineWidth">
    <number>4</number>
   </property>
   <property name="text">
    <string> s'inscrire</string>
   </property>
   <property name="textFormat">
    <enum>Qt::TextFormat::AutoText</enum>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <property name="openExternalLinks">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>210</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="tabletTracking">
    <bool>false</bool>
   </property>
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(9, 130, 230);
background-color: qlineargradient(spread:repeat, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(9, 130, 230 ,10) stop:1 rgba(255, 255, 255, 255));
border-radius:20px;
font-family: Lexend;
border-radius:5px;
color:#001F54;  /* Navy Blue */

font-size:20px;
border-color: #001F54;  /* Navy Blue */

border-style:20px solid #0982e6;;


QLineEdit
{
background:transparent;
border:none;
color:#0982e6;
border.bottum:1px solid #0982e6;
border-radius:25px;
}</string>
   </property>
   <property name="text">
    <string>identifiant</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::EchoMode::NoEcho</enum>
   </property>
   <property name="dragEnabled">
    <bool>false</bool>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="cursorMoveStyle">
    <enum>Qt::CursorMoveStyle::LogicalMoveStyle</enum>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="add_employee_btn_4">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>460</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
border-radius:25px;
color:#001F54;  /* Navy Blue */
font-family: Lexend;
font-size:16px;
border-color: #001F54;  /* Navy Blue */
border-width : 1.2px;
border-style:solid;
background-color:white;
background-color: qlineargradient(spread:repeat, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(9, 130, 230 ,10) stop:1 rgba(255, 255, 255, 255));
}
QPushButton:hover
{
background:white;
color:#001F54;  /* Navy Blue */
border-left:3px solid #00b521 ; 
border-color:none;
border-radius:25px;
background-color: qlineargradient(spread:repeat, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(9, 130, 230 ,10) stop:1 rgba(255, 255, 255, 255));
}
QPushButton:pressed
{
background:#013220;
color:#001F54;  /* Navy Blue */
border-radius:25px;
font-family: Lexend;
font-size:16px;
border-left:3.2px solid #1BA019 ; 
/*background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                stop: 0 #ffffff, stop: 1 #dcdcdc);*/}

</string>
   </property>
   <property name="text">
    <string>face ID </string>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/images/add-circle.png</normalon>
    </iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>200</width>
     <height>200</height>
    </size>
   </property>
   <property name="autoRepeatInterval">
    <number>100</number>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>label_3</zorder>
  <zorder>lineEdit</zorder>
  <zorder>motdepasseoublier_2</zorder>
  <zorder>add_employee_btn_3</zorder>
  <zorder>add_employee_btn_4</zorder>
  <zorder>label_2</zorder>
  <zorder>lineEdit_2</zorder>
  <zorder>motdepasseoublier</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
